src/main/resources/templates/placeOrder.html

```diff
@@ -20,7 +20,7 @@ <h1 class="display-4">Place an Order</h1>
 </main>
 
 <div class="container">
-    <form>
+    <form id="placeOrder" method="post" action="">
```

Comment:
You want to make sure forms, and anything with input has an id.  Also, ensure your form has an HTTP method.

Reply:
It should probably have an action, too.  :)